Helpers/PriceQueryHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Helpers;
+
+use App\Models\Price;
+use App\Helpers\TenantDbHelper;
+
+class PriceQueryHelper
```

Comment:
This is an example of how extensive logic in the controller can be outsourced to an external helper class. This is more of an theoretical proposal it could be that Laravel has a slightly different preferred convention on how to encapsulate such logic.